components/google-cloud/storage/download/component.yaml

```diff
@@ -1,8 +1,8 @@
 name: Download from GCS
 inputs:
-- {name: GCS path, type: URI}
+- {name: GCS path, type: String}
 outputs:
-- {name: Data}
+- {name: Data, type: Artifact}
```

Comment:
This change makes the component incompatible with most downstream components. 
Type `Artifact` is not compatible with the type `XGBoostModel`, `PyTorchModel` etc.

Reply:
Ok, I reverted this change.